test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -347,14 +356,15 @@ public void TestRunErrorMessageShowShouldTestRunFailed()
         }
 
         [TestMethod]
-        public void InQuietModeTestErrorMessageShowShouldShowTestRunFailed()
+        public void InQuietModeTestErrorMessageShouldShowTestRunFailed()
         {
             // Setup
             var loggerEvents = new InternalTestLoggerEvents(TestSessionMessageLogger.Instance);
             loggerEvents.EnableEvents();
             var parameters = new Dictionary<string, string>
             {
-                { "verbosity", "quiet" }
+                { "verbosity", "quiet" },
+                { DefaultLoggerParameterNames.TargetFramework , "abc" }
```

Comment:
Why do we need this change ? I thought we accept framework as null

Reply:
yes, we accept.